### PR TITLE
Edit User Guide to include all the must-have commands

### PR DESCRIPTION
**Pull request by Mamtha3005**
Edited User Guide with all the must-have commands and relevant pictures.
@CS2103AUG2016-W15-C3/documentation Ready For Review

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,128 +1,152 @@
...
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The parameters have to be in the given order below.
 
-#### Viewing help : `help`
-Format: `help`
+#### Viewing list of commands : `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+When unsure of the available commands or command format, Enter help.
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Please markup <kbd>Enter</kbd>

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,128 +1,152 @@
...
-   * **`delete`**` 3` : deletes the 3rd task shown in the current list
-   * **`exit`** : exits the application
-6. Refer to the [Features](#features) section below for details of each command.<br>
+Step 2: Download the latest `taskell.jar` from here.<br>
+	 <img src="images/Icon.png" width="100"><br>
+Step 3: Copy the file to the folder you want to use as the home folder for your Task Manager.<br>
+Step 4: Double-click the file to start the application. The GUI should appear in a few seconds. <br>
+ <img src="images/GUI.png" width="600"><br>
+Picture 1: A screenshot of the Graphical User Interface (GUI)<br>
+Step 5:	Type the relevant command in the command box and press enter to execute it.<br>
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Please markup <kbd>enter</kbd>

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,128 +1,152 @@
...
+-  `add TASK ITEM on [DATE]` <br>
+The use of word �on� gives more flexibility for you.<br>
+> Example: `add schedule meeting on Thursday` <br>
+- `add TASK ITEM on [DATE][TIME]` <br>
+> Example: `add meet teacher on Friday 7pm` <br>
+
+Take Note! Dates need to refer to the current week dates the task was keyed in. Eg. "tomorrow" and "thursday" is valid but "next Saturday" is invalid. Only the words "on" or "by" has to be used to distinguish between dateline and task description.<br>
+
+#### Listing all tasks : `list`
+To view a list of all the tasks, <kbd>Enter<kbd> list.<br>
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Incorrect closing tag after <kbd>Enter</kbd>

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,128 +1,152 @@
...
+   
+#### Finding tasks: `find`
+To view tasks with similar keyword, use the find command.<br>
+Formats: <br>
+-`find KEYWORD [MORE_KEYWORDS]`<br>
+Prints a list of activities that match the keywords.<br>
+Example: `find banana milk essay: returns any task description having keywords banana, milk, essay.` <br>
+-`find-tag KEYWORD `<br>
+Prints list of activities with the same tag.<br>
+ <img src="images/findReport.png" width="600">
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Mismatching file naming. Please correct the file name in images.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,128 +1,152 @@
...
+- `add TASK ITEM by [DATE][TIME]` <br>
+This format can be used to add tasks with a stipulated deadline.<br>
+> Example: `add do lab homework on Friday 7pm` <br>
+-  `add TASK ITEM on [DATE]` <br>
+The use of word �on� gives more flexibility for you.<br>
+> Example: `add schedule meeting on Thursday` <br>
+- `add TASK ITEM on [DATE][TIME]` <br>
+> Example: `add meet teacher on Friday 7pm` <br>
+
+Take Note! Dates need to refer to the current week dates the task was keyed in. Eg. "tomorrow" and "thursday" is valid but "next Saturday" is invalid. Only the words "on" or "by" has to be used to distinguish between dateline and task description.<br>
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Missing markup for callouts

**Comment by marcusngwenjian** [comment]
@Mamtha3005 
Reviewed. Kindly make the changes.

**Comment by Mamtha3005** [comment]
Reviewed with changes made.
@CS2103AUG2016-W15-C3/integration Ready for integration.

**Comment by Mamtha3005** [comment]
Changes made and reviewed again. @CS2103AUG2016-W15-C3/integration ready for integration.

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/documentation 
1. Would be good to have linebreak before each picture insert, to make use of whitespace
2. Refer command summary: inconsistent formatting i.e. "List Given Day" and "Add event with deadline", each first letter all caps or only cap first word's first letter?
3. Saving info does not have command format

**Comment by Mamtha3005** [comment]
@CS2103AUG2016-W15-C3/integration Changes made. Ready for review.

**Pull request merged**